cloudify_agent/api/pm/base.py

```diff
@@ -255,8 +259,9 @@ def __init__(self, logger=None, **params):
         self.rest_protocol = params.get(
             'rest_protocol') or defaults.REST_PROTOCOL
         self.security_enabled = params.get('security_enabled')
-        self.rest_username = params.get('rest_username')
-        self.rest_password = params.get('rest_password')
+        self._rest_username = params.get('rest_username')
```

Comment:
making these properties "hidden" (`_`) is inconsistent with the other properties